doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -646,9 +646,6 @@ Response:
 * Successful operation uses 204 No Content
 * Errors use a non-200 and (optionally) an error payload. See subsection on Error Responses for more details.
 
-+ Parameters
```

Comment:
This has not been "recoved" from the resource level to the verb level (i.e. the APIary render doesn't show it at the right panel as required parameter).